.env.example

```diff
@@ -8,4 +8,5 @@ GOOGLE_CLIENT_SECRET=""
 AZURE_CLIENT_ID=""
 AZURE_CLIENT_SECRET=""
 AZURE_TENANT="consumers"
-HOST_URL="http://localhost:3000"
\ No newline at end of file
+HOST_URL="http://localhost:3000"
+TZ="UTC"
```

Comment:
Maybe 🤷 
I would assume that the default for the server is UTC, but it could maybe be GMT instead 😅 

We should deploy to staging before production